### PR TITLE
8.2.0

### PR DESCRIPTION
**Pull request by mattrobenolt**
TODO:
- [x] Update config files

### CODE REVIEW

File: 8.2/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,21 @@
-FROM python:2.7
+FROM python:2.7-slim
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        curl \
+        gcc \
+        git \
```
Comment (rassie) [inline 8.2/Dockerfile]: Do you have plans on enabling `dSym` support? My colleagues are pretty hyped about that and the [original branch merge](https://github.com/getsentry/sentry/commit/ed471f075da35a4edd4ef547595a4cbb9feadf59#diff-08bc83c6a89d6b335935cbb22c40f36eR29) adds `clang` and `g++` to the `Dockerfile`.

Comment (mattrobenolt) [inline 8.2/Dockerfile]: I think the plan is for 8.3. In 8.2 at the moment, it's not exposed and hasn't been tested by us much yet.

So hang in there, it'll come soon. :)

Comment (rassie) [inline 8.2/Dockerfile]: But I suppose there is nothing stopping me from abusing `-onbuild` to enable it myself, right? ;)

Comment (mattrobenolt) [inline 8.2/Dockerfile]: Yeah, totally. :)

**Pull request merged**